recipes/newrelic/infrastructure/ohi/mysql/debian.yml

```diff
@@ -85,7 +85,7 @@ install:
             exit 1
           fi
         - |
-          CHECK_DB=$(mysql -u{{.NR_CLI_DB_USERNAME}} --port {{.NR_CLI_DB_PORT}} -p{{.NR_CLI_DB_PASSWORD}} -e "SHOW GRANTS FOR CURRENT_USER" | grep 'GRANT SELECT' | awk '{print $2, $3}')
+          CHECK_DB=$(mysql -u{{.NR_CLI_DB_USERNAME}} --port {{.NR_CLI_DB_PORT}} -p{{.NR_CLI_DB_PASSWORD}} -e "SHOW GRANTS FOR CURRENT_USER" 2>&1 | grep -v "mysql: [Warning] Using a password on the command line interface can be insecure." | grep 'GRANT ALL\|SELECT' | awk '{print $2, $3}')
```

Comment:
@gabeobrien @Julien4218 In the course of doing the `assert_pre_req` task, we call mysql from the command line and validate the user's grants. That causes this message to be displayed:

`mysql: [Warning] Using a password on the command line interface can be insecure.`

I added this code to suppress that message from the user. Does that seem appropriate or should we let the warning fly?

![image](https://user-images.githubusercontent.com/6722433/107831158-6b9f8500-6d5b-11eb-9df4-6f3018b4ff66.png)

Reply:
This is done in the rhel and suse recipes as well.

Reply:
Seems like suppression the message from MySQL is a good call, since it displays awkwardly (no line break) and we do not have control over what the message is.
Could we add our own message that mimics the warning?  That way we surface an issue that the user might care about, but we control what that message is and how it is displayed.

Reply:
Since we have control of stdout in the recipe, we could do something like:

```
Verifying MySQL requirements...
Checks passed.
```